test/integration/pytest_itde_test.py

```diff
@@ -1,2 +1,142 @@
-def test_smoke_test_plugin(itde):
-    assert True
+from inspect import cleandoc
+from itertools import chain
+
+import pytest
+
+pytest_plugins = "pytester"
+
+
+@pytest.fixture
+def make_test_files():
+    def make(pytester, files):
+        pytester.makepyfile(**files)
+
+    return make
+
+
+def _ids(params):
+    keys = (k for k in params.keys())
+    return next(keys)
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "files",
+    [
+        {
+            "test_smoke_test": cleandoc(
+                """
```

Comment:
Kind of crazy, that we create and run tests with pytest in a test which runs pytest. Inception. However, a good test.